config-linux.md

```diff
@@ -214,9 +214,9 @@ Runtimes MAY attach the container process to additional cgroup controllers beyon
 }
 ```
 
-### <a name="configLinuxDeviceWhitelist" />Device whitelist
```

Comment:
Since "whitelist" implies a deny-by-default configuration (with "blacklist" implying the reverse), we should add some text to clarify that this is the case. At least to my ear, "allowlist" doesn't have that implication (at least not as apparently as "whitelist").

Reply:
block/unblock?

Reply:
I should've better clarified my point here -- the title "device allowlist" is totally fine (and I prefer it to "allowed device list"). What I was suggesting is to add a single sentence which says something like:

> The allow-list MUST have a default deny-all policy, meaning that if it is omitted no device access is permitted.

This would not be a change in behaviour (since that is how all implementations have worked and is technically already required by the term "whitelist") but instead just a clarification.